src/integrationTest/java/uk/gov/hmcts/darts/testutils/data/MediaTestData.java

```diff
@@ -71,7 +66,7 @@ public static MediaEntity createMediaWith(CourtroomEntity courtroomEntity, Offse
     }
 
     private String getChecksum() {
-        return new String(encodeBase64(md5(getBinaryData().toBytes())));
+        return TestUtils.encodeToString(md5(MEDIA_TEST_DATA_BINARY_DATA));
```

Comment:
You can call the FileContentChecksum instead so there aren't multiple versions of doing the same thing

Reply:
that method is `private`, I would have to make it `public` (not even `protected` will work) just for purpose of testing. Is that an acceptable tradeoff?